module/spock/spock_verifier_test.go

```diff
@@ -0,0 +1,254 @@
+package spock
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/crypto/hash"
+	"github.com/onflow/flow-go/model/encoding"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/local"
+	realproto "github.com/onflow/flow-go/state/protocol"
+	protocol "github.com/onflow/flow-go/state/protocol/mock"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// SpockVerifierTestSuite contains tests against methods of the SpockVerifier scheme
+type SpockVerifierTestSuite struct {
+	suite.Suite
+
+	// map to hold test idenitities
+	identities map[flow.Identifier]*flow.Identity
+
+	snapshot *protocol.Snapshot
+	state    *protocol.State
+
+	exeLocal *local.Local
+	verLocal *local.Local
+
+	spockHasher hash.Hasher
+
+	verifier *Verifier
+}
+
+// TestSpockVerifer invokes all the tests in this test suite
+func TestSpockVerifer(t *testing.T) {
+	suite.Run(t, new(SpockVerifierTestSuite))
+}
+
+// Setup test with n verification nodes
+func (vs *SpockVerifierTestSuite) SetupTest() {
```

Comment:
I think `godoc` needs a revision.

Reply:
Also, just as a suggestion, I think if you rename the receiver variable of test suite methods to `suite`, it adds more readability.